operator/src/main/resources/scripts/traceUtils.sh

```diff
@@ -164,3 +164,107 @@ function exportEffectiveDomainHome() {
   trace "Error: More than one config.xml found at DOMAIN_HOME/config/config.xml and DOMAIN_HOME/*/config/config.xml, DOMAIN_HOME='$DOMAIN_HOME': ${found_configs}. Configure your 'domainHome' setting in your WebLogic Operator Domain resource to reference a single WebLogic domain."
   return 1
 }
+
```

Comment:
Should we rename traceUtils.sh now that it is not just for trace utilities? Or perhaps we should update the "Purpose" section in the beginning of the file.

Reply:
Good point.  I'll go ahead and do both.

Reply:
I've made the changes, please resolve if you're fine with it.